core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/AvroUpsertMonitor.scala

```diff
@@ -233,15 +238,23 @@ class AvroUpsertMonitorActor(
     val attributes = parseMessage(message)
 
     val importFuture = for {
-      petUserInfo <- getPetServiceAccountUserInfo(attributes.workspace.namespace, attributes.userEmail)
+      workspace <- dataSource.inTransaction { dataaccess =>
+        dataaccess.workspaceQuery.findByName(attributes.workspace).map {
+          case Some(workspace) => workspace
+          case None =>
+            publishMessageToUpdateImportStatus(attributes.importId, None, ImportStatuses.Error, Option(s"Workspace ${attributes.workspace} not found"))
+            throw new RawlsException(s"Workspace ${attributes.workspace} not found while importing entities")
+        }
+      }
```

Comment:
@davidangb could I get your eyes here? The google project can no longer be assumed to be the workspace namespace so we need to lookup the workspace in the db.

Reply:
this looks right to me!